File: scripts/maintenance/separate_containers.py
with open(r'c:\Users\PC2\Desktop\AOAS WEB\careers.html', 'r', encoding='utf-8') as f:
    content = f.read()

# Replace the map section placement
old_text = '''            </form>

            <!-- Map Section -->
            <div class="map-section">'''

new_text = '''            </form>
        </div>

        <!-- Map Section -->
        <div class="form-container" style="margin-top: 32px;">
            <div class="map-section">'''

content = content.replace(old_text, new_text)

with open(r'c:\Users\PC2\Desktop\AOAS WEB\careers.html', 'w', encoding='utf-8') as f:
    f.write(content)

print("Successfully separated the form and map containers!")

File: scripts/maintenance/separate_containers.js
const fs = require('fs');

// Read the HTML file
let content = fs.readFileSync('careers.html', 'utf-8');

console.log("Before replacement:");
const checkText = content.substring(content.indexOf('</form>'), content.indexOf('</form>') + 100);
console.log(checkText.substring(0, 100));

// First replacement: close form-container after form
content = content.replace(
    '            </form>\r\n\r\n            <!-- Map Section -->',
    '            </form>\r\n        </div>\r\n\r\n        <!-- Map Section -->'
);

// Second replacement: wrap map section in new form-container
content = content.replace(
    '        </div>\r\n\r\n        <!-- Map Section -->\r\n            <div class="map-section">',
    '        </div>\r\n\r\n        <!-- Map Section -->\r\n        <div class="form-container" style="margin-top: 32px;">\r\n            <div class="map-section">'
);

console.log("\nAfter replacement:");
const checkText2 = content.substring(content.indexOf('</form>'), content.indexOf('</form>') + 200);
console.log(checkText2.substring(0, 200));

// Write the modified content back
fs.writeFileSync('careers.html', content, 'utf-8');

console.log('\n✓ Successfully separated the form and map containers!');
